frontend/src/app/supportAdmin/DeviceType/__snapshots__/ManageDevicesForm.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ManageDeviceTypeForm renders the Manage Device Type Form Container item 1`] = `undefined`;
```

Comment:
curious - what does this do? is the snapshot saved elsewhere?

Reply:
If the snapshot changes the test will fail until you update the snapshot.  At least for me it just serves as a reminder to have someone from design look at the new UI changes.